src/browser/CameraProxy.js

```diff
@@ -102,7 +102,11 @@ function capture (success, errorCallback, opts) {
 
     var successCallback = function (stream) {
         localMediaStream = stream;
-        video.src = window.URL.createObjectURL(localMediaStream);
+        try {
```

Comment:
I would do property detection instead of a try/catch.
```
if ('srcObject' in video) {
   // 
} else {
   //
}
```

Reply:
Updated, thanks @shazron